--- origsrc/libvpx-v0.9.5/build/make/configure.sh	2010-10-28 08:14:14.000000000 -0500
+++ src/libvpx-v0.9.5/build/make/configure.sh	2011-01-16 16:30:33.709741100 -0600
@@ -877,7 +877,7 @@ process_common_toolchain() {
         [ "${AS##*/}" = nasm ] && add_asflags -Ox
         AS_SFX=.asm
         case  ${tgt_os} in
-            win*)
+            win*|cygwin*)
                 add_asflags -f win${bits}
                 enabled debug && add_asflags -g dwarf2
             ;;
--- origsrc/libvpx-v0.9.5/configure	2010-10-28 08:14:14.000000000 -0500
+++ src/libvpx-v0.9.5/configure	2011-01-16 16:30:56.729772800 -0600
@@ -102,6 +102,7 @@ all_platforms="${all_platforms} ppc64-da
 all_platforms="${all_platforms} ppc64-darwin9-gcc"
 all_platforms="${all_platforms} ppc64-linux-gcc"
 all_platforms="${all_platforms} sparc-solaris-gcc"
+all_platforms="${all_platforms} x86-cygwin-gcc"
 all_platforms="${all_platforms} x86-darwin8-gcc"
 all_platforms="${all_platforms} x86-darwin8-icc"
 all_platforms="${all_platforms} x86-darwin9-gcc"
